### PR TITLE
Webauthn

### PR DESCRIPTION
**Pull request by neurosnap**
This PR does two things:
- Fixes a bug with logging in via webauthn
- Ability to register security keys using app-ui

### CODE REVIEW

File: src/ui/pages/login.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ export const LoginPage = () => {
       dispatch(
         loginWebauthn({
           ...loginPayload,
-          webauthn: loader.meta.exception_context.u2f,
+          webauthn: loader.meta.exception_context.u2f?.payload,
```
Comment (neurosnap) [inline src/ui/pages/login.tsx:91]: This was the bug

Comment (madhuravius) [reply src/ui/pages/login.tsx:91]: 😠 wow

is there an easy way to test this btw along with some of the other flows when we're closer to ready?

Comment (madhuravius) [reply src/ui/pages/login.tsx:91]: also i can't see the types on this pr for `loginWebauthn`, shouldn't that ideally have saved us?

Comment (neurosnap) [reply src/ui/pages/login.tsx:91]: The types are created by us so we if we don't type things properly then you can get a runtime error.

Something like `zod` probably could have saved us here.

Comment (madhuravius) [reply src/ui/pages/login.tsx:91]: ✅  - tested on sbx main
looks like an error on typing on OUR end pre-send to auth api

**Review by madhuravius — APPROVED** [review]

**Pull request merged**